wsd/Admin.hpp

```diff
@@ -248,6 +252,8 @@ class Admin : public SocketPoll
 
     // map to make sure only connection with unique monitor uri exists
     std::map<std::string, std::shared_ptr<MonitorSocketHandler>> _monitorSockets;
+
+    bool _closeMonitor = false;
```

Comment:
Technically, this should be `atomic_bool`. Same for `_migrateMsgReceived`.

Reply:
> Technically, this should be `atomic_bool`. Same for `_migrateMsgReceived`.

If we make _migrateMsgReceived atomic bool do we need this lock  ? 
https://github.com/CollaboraOnline/online/pull/8679/files#diff-e65ca5973009b9c5a78569b89969010bfa8b8459e26ca0a251f89f49a08a3afbR3320

Reply:
Absolutely! Changing `_migrateMsgReceived` (which is a new variable) doesn't change existing `DocBrokers` and the need for the lock. That lock is to protect `DocBrokers`.